docs/getting-started/installation.md

```diff
@@ -216,15 +218,15 @@ pip3 install prefect
 
 ## Installing unreleased code
 
-To use the most up-to-date, unreleased Prefect code, you can install directly off the `main` GitHub branch:
+To use the most up-to-date, unreleased Prefect 2 code, you can install directly off the `2.x` GitHub branch:
 
 <div class="terminal">
 ```bash
-pip install -U git+https://github.com/PrefectHQ/prefect
+pip install --branch 2.x -U git+https://github.com/PrefectHQ/prefect
```

Comment:
same as above

Reply:
Removing this section as users can find in the contributing section.